Expensify/react-native-live-markdown
# try fix 41908

### Details


### Related Issues
$ https://github.com/Expensify/App/issues/41908
proposal: https://github.com/Expensify/App/issues/41908#issuecomment-2114919703

### Manual Tests
#### Test in react-native-live-markdown
1. `yarn install`
2. Start examples:
    - For web: 
        + From project root
        + `cd WebExample`
        + `npm run web`
    - For android:
        + From project root
        + `yarn example start`
        + `yarn example android`
    - For ios:
        + From project root
        + `yarn example start`
        + `yarn example ios`
3. Input `   # header`, verify header style ok
4. Input `    > quote content`, verify quote style ok

#### Test in expensify app
0. Prepare react-native-live-markdown
    - in react-native-live-markdown project root
    - `yarn install`
    - `npm run prepare`
1. Switch to expensify app project root
2. Checkout latest main branch
3. `npm install`
4. Create a `tmp` file
5. Copy following content to `tmp` file:
    ```
    cd node_modules/@expensify/react-native-live-markdown
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/LICENSE ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/README.md ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/RNLiveMarkdown.podspec ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/android ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/cpp ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/ios ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/lib ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/package.json ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/parser ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/react-native.config.js ./
    cp -r /Users/zhaoxuxu/expensify/react-native-live-markdown/src ./
    ```
6. Edit `tmp` file, change `/Users/zhaoxuxu/expensify` to `your path to react-native-live-markdown`
7. `bash tmp` to copy built live-markdown files to expensify-app dependency
8. Start expensify-app to verify

### Linked PRs
Need change [package.json version of expensify app](https://github.com/Expensify/App). No pr created yet, need build react-native-live-markdown by hand and copy related files to node_modules of Expensify-App to verify.

### Video evidences
ios:

https://github.com/Expensify/react-native-live-markdown/assets/10908996/0570fc6a-5867-4a0c-9061-8b7308ef546b

android:

https://github.com/Expensify/react-native-live-markdown/assets/10908996/cf3b9999-c972-4975-95c3-ae107d4662d3

web:

https://github.com/Expensify/react-native-live-markdown/assets/10908996/f05423e5-a086-4bfe-8e4e-405d7e2115bd

ios-expensify:

https://github.com/Expensify/react-native-live-markdown/assets/10908996/b78f7ae9-5585-44a1-a9c7-01ae33c4ff23

android-expensify:

https://github.com/Expensify/react-native-live-markdown/assets/10908996/3f1128cb-e3ba-4bcc-9980-806da8262e6d

web-expensify:

https://github.com/Expensify/react-native-live-markdown/assets/10908996/0b8d50d1-6d60-48fe-8cc0-7bf57f57c755
